src/Cache/FileCache.php

```diff
@@ -179,7 +179,13 @@ protected function removeEmptyDirectories(string $dir): void
 
 			// checks all directory segments until reaching the root directory
 			while (Str::startsWith($dir, $this->root()) === true && $dir !== $this->root()) {
-				$files = array_diff(scandir($dir) ?? [], ['.', '..']);
+				$files = scandir($dir);
```

Comment:
`scandir()` return isn't null but false, so fallback to empty array has to be handled differently